tests/e2e/tests/test/test.ts

```diff
@@ -1,7 +1,10 @@
 import { ng } from '../../utils/process';
+import { copyFile } from '../../utils/fs';
 
 export default function () {
   // make sure both --watch=false and --single-run work
   return ng('test', '--single-run')
-    .then(() => ng('test', '--watch=false'));
+    .then(() => ng('test', '--watch=false'))
+    .then(() => copyFile('./karma.conf.js', './karma.conf.bis.js'))
```

Comment:
Use moveFile instead of copy please, otherwise the feature might be broken and we don't know since the old file is used.

The test harness will take care of moving it back after the test.